webapp/src/components/DonationsDashboard/DonationsDashboard.js

```diff
@@ -398,8 +403,12 @@ const DonationsDashboard = ({ isDesktop, currentUser, isOffer }) => {
   }, [currentUser, history, client, loadProfile, state, open, transferResult])
 
   useEffect(() => {
-    setTokens(role === "donor" && profile?.balance.length ? profile.balance.join(',').split(' ')[0] : 0)
-  }, [profile])
+    if(tokenUser.user) {
```

Comment:
When it just one single line in the body of a comparison, you can omit the curly braces

Reply:
@JustinCast I'm just following this standard, is it right or I should to change it?

![image](https://user-images.githubusercontent.com/28828796/111375688-2d4ffb00-8664-11eb-8e07-cb987f79149c.png)

Reply:
@leisterfrancisco you can use the first one